src/test/docker/docker-run-checks.sh

```diff
@@ -59,9 +59,14 @@ Options:\n\
 "
 
 # check if running in OSX
-if [[ "$(uname)" == "Darwin" ]]; then
+if [[ "$(uname)" == "Darwin" ]] && [[ $FORCE_GNU_GETOPT != 1 ]]; then
     # BSD getopt
     GETOPTS=`getopt $short_opts -- $*`
+    export EXTRA_MACOS_STUFF="\n\
+    You are using BSD getopt on macOS. BSD getopt does not recognize '='\n\
+    between options. Use a space instead. If gnu-getopt is first in your\n\
+    PATH, force the script to use that by setting FORCE_GNU_GETOPT=1.\n"
+    usage=${usage}${EXTRA_MACOS_STUFF}
```

Comment:
It shouldn't be necessary to use `export` here since `EXTRA_MACOS_STUFF` does not need to be exported to the environment of future commands. How about just:
```sh
   usage="${usage}\n\
   You are using BSD getopt on macOS. BSD getopt does not recognize '='\n\
   between options. Use a space instead. If gnu-getopt is first in your\n\
   PATH, force the script to use that by setting FORCE_GNU_GETOPT=1.\n"
```

Reply:
Good catch @grondo, fixed.